kvbc/tools/db_editor/include/kv_blockchain_db_editor.hpp

```diff
@@ -301,6 +301,93 @@ struct GetCategories {
   }
 };
 
+struct GetEarliestCategoryUpdates {
+  const bool read_only = true;
+  std::string description() const {
+    return "getEarliestCategoryUpdates CATEGORY-ID [BLOCK-VERSION-TO]\n"
+           "  Returns the latest blockID that contains the given category, started from BLOCK-VERSION-TO. If "
+           "BLOCK-VERSION-TO is not set, we start from the lastReachableBlock.";
```

Comment:
I think the description is confusing, because we say `earliest` and then `latest blockID`. Also, we say `started from`, but the block is `TO`.

Please correct me if I am wrong, but I think it can be described similar to:

```
Returns the first blockID that contains the given category in the [genesisBlockID, BLOCK-VERSION-TO] range.
If BLOCK-VERSION-TO is not set, the search range is [genesisBlockID, lastReachableBlockID].
```